README.md

```diff
@@ -17,3 +17,86 @@ The production version runs on Amazon Elastic Beanstalk (AWS EB) and therefore t
 **server.js** is the primary file launching the API daemon.
 
 **routes/*.js** contains all the endpoints for the API service. Currently, only one exists for weather adjustment.
+
+---
+#### Local Installation onto a Raspberry Pi
+
+Step 1: Download and install Node.js onto the Raspberry Pi so that we can run the OpenSprinkler weather server. The version of Node.js to install is dependent on your model of Raspberry Pi.
+
+Note: you can run the command ```uname -m``` on your Raspberry Pi to help identify the chipset that is being used.
+
+*For Raspberry Pi 2 or Pi 3 models that are based on the newer ARMv7 and ARMv8 chip*
+```
+pi@OSPi:~ $ curl -sL https://deb.nodesource.com/setup_11.x | sudo -E bash -l
+pi@OSPi:~ $ sudo apt install -y nodejs
+```
+
+*For Raspberry Pi Model A, B, B+, Zero and Compute Module based on the older ARMv6 chip, the process is slightly more convoluted*
+```
+pi@OSPi:~ $ wget https://nodejs.org/dist/v11.4.0/node-v11.4.0-linux-armv6l.tar.gz 
+pi@OSPi:~ $ tar -xvf node-v11.4.0-linux-armv6l.tar.gz 
+pi@OSPi:~ $ cd node-v11.4.0-linux-armv6l
+pi@OSPi:~ $ sudo cp -R * /usr/local/
+pi@OSPi:~ $ cd ..
+pi@OSPi:~ $ rm -rf node-v11.4.0-linux-armv6l
+pi@OSPi:~ $ rm node-v11.4.0-linux-armv6l.tar.gz
+
+```
+
+Step 2: Download the OpenSprinkler Weather Service repository to your Raspberry Pi so that we can run a local version of the service:
+
+```
+pi@OSPi:~ $ git clone https://github.com/OpenSprinkler/OpenSprinkler-Weather.git weather
+```
+
+Step 3: Install all of the dependencies using the Node Package Manager, npm, from within the weather project directory:
+```
+pi@OSPi:~ $ cd weather
+pi@OSPi:~/weather $ npm install
+```
+Step 4: The file .env is used by the weather server to specify the interface and port to listen on for OpenSprinkler Firmware weather requests. We need to create a new file, .env, and enter some configuration details.
+```
+pi@OSPi:~/weather $ nano .env
+```
+
+Add the following two lines to the .env file so that the weather server is configured to listen for weather requests. Using 0.0.0.0 as the host interfaces allows you to access the service from another machine to test. Alternatively, set HOST to “localhost” if you want to limit weather service access to only applications running on the local machine.
+
```

Comment:
In this section, should we add the OWM key as an optional for people not configuring it for use with a local weather station?

Reply:
Yep, good call.